app/models/game.rb

```diff
@@ -22,5 +23,45 @@ def coordinate_conflict?(test_coords)
     false
   end
 
+  def populate_game
+    # populate black pieces at top
+  
+    (0..7).each do |n|
+      pieces.create(type: "Pawn", color: "black", x_pos: n, y_pos: 6)
```

Comment:
Thanks. That's how I initially tried to set it up, but I couldn't get it to work - which is why out of desperation, I added type. I couldn't figure out why it wouldn't work with STI. Of course, I left out game: self. Making that change now.